.github/PULL_REQUEST_TEMPLATE/python.md

```diff
@@ -0,0 +1,169 @@
+**Changes:**
+
+* IoT3 SDK:
+    * core: ...
+    * mobility: ...
+* applications:
+    * iqm: ...
+    * ...
+
+Closes: #
+
+---
+**How to test:**
+
+_**Note:** in the following, lines starting with `$` are to be executed on your machine, as a non-root user; lines starting with `(docker)$` are to be executed in the Docker container; lines starting with `(docker)🐍 $` are to be executed in the Docker container, in the python venv._
+
+1. Prepare a test environment:
+    1. be sure to have unrestricted access to _test.mosquitto.org_ (IPv4 and IPv6)
+    2. be sure to have an MQTT broker that listens locally on port 1883, with no credentials and no ACL; if not, run your own:
+        ```sh
+        $ mosquitto
+        ```
+    3. in another terminal, prepare an OpenTelemetry collector on localhost:
+        ```
+        $ docker container run \
+            --rm \
+            -p 16686:16686 \
+            -p 4318:4318 \
+            jaegertracing/all-in-one:1.58
+        ```
+        then open a browser on the Jaegger UI:
+        ```
+        http://localhost:16686/
+        ```
+    4. in another terminal, start a fake _gpsd_ process (no need for an actual GNSS device):
+        ```sh
+        $ TMPDIR=/tmp gpsfake -q -P 2947 -u -n -c 0.1 tests/data/NMEA.log
+        ```
+    5. in another terminal, start a container with python 3.11 and the necessary packages:
```

Comment:
Python?

Reply:
You mean just uppercasing _Python_, right?